src/foremast/consts.py

```diff
@@ -210,6 +210,7 @@ def _generate_security_groups(config_key):
 APP_FORMATS = extract_formats(config)
 GATE_CLIENT_CERT = expandvars(expanduser(validate_key_values(config, 'base', 'gate_client_cert', default='')))
 GATE_CA_BUNDLE = expandvars(expanduser(validate_key_values(config, 'base', 'gate_ca_bundle', default='')))
+LINKS = _convert_string_to_native(validate_key_values(config, 'urls', 'spinnaker_links', {}))
```

Comment:
it might make more sense for the group to be named something like `default_links` instead of `urls`. We have other URLs elsewhere in the config.